src/client/components/common/button/button.jsx

```diff
@@ -12,20 +10,18 @@ const colors = {
   red: 'red',
 };
 
-type ColorType = 'green' | 'blue' | 'red';
-
-type PropsType = {
-  children: Node,
-  onClick?: (e: SyntheticEvent<HTMLDivElement>) => Promise<*> | void,
-  onKeyDown?: (e: SyntheticKeyboardEvent<HTMLDivElement>) => Promise<*> | void,
-  tabIndex?: string | number,
-  color?: ColorType,
-  className?: string,
-};
-
 const noop = () => {};
 
-class Button extends Component<PropsType> {
+class Button extends Component {
+  static propTypes = {

```

Comment:
Please rewrite `propTypes` to be outside the components:
e.g. `Button.ptopTypes = {
  //...
}`
because [other paralect repos](https://github.com/paralect/nextjs-landing-starter/blob/bc108dfcf1b4b85e49f2542e1ef1b20e1bfa1db5/src/client/components/button/button.jsx) are using such propTypes notation